packages/sftp-client/index.js

```diff
@@ -134,12 +134,10 @@ class SftpClient {
    * @returns {Promise<Array<Object>>} list of file objects
    */
   async list(remotePath) {
-    const actualPath = remotePath.trim() === '' ? '/' : remotePath;
```

Comment:
So if the `removePath` was empty, start at `/`. Is this behavior preserved?

Reply:
This was a fix Marc put in after I merged my previous PR. It has been consolidated into `recursion.js` which uses `path.normalize` to ensure there are never empty paths passed down the chain.